toutvcli/tests/test_app_real.py

```diff
@@ -56,19 +57,23 @@ def testSearch(self):
 
     def _testFetch(self, expected_filename, args):
         # Delete the file if it exists prior to the test.
-        if os.path.exists(expected_filename):
-            os.unlink(expected_filename)
+        for file in glob.glob(expected_filename):
+            os.unlink(file)
 
         args = ['--verbose', 'fetch', '-qMIN'] + args
 
         self.assertEqual(app.App(args).run(), 0)
 
-        self.assertTrue(os.path.exists(expected_filename))
+        file_exists = False;
```

Comment:
To avoid iterating on the list, you could do
```python
file_exists = len(glob.glob(expected_filename)) > 0
```
or even directly
```
self.assertTrue(len(glob.glob(expected_filename)) > 0)
```
But I'm not against the intermediate variable, both ways are fine.